Orange/widgets/unsupervised/owdistances.py

```diff
@@ -58,7 +58,7 @@ class MetricDef(NamedTuple):
                   distance.PearsonRAbsolute),
         MetricDef(Spearman, "Spearman",
                   "Pearson correlation; distance = 1 - ρ/2",
-                  distance.PearsonR),
+                  distance.SpearmanR),
```

Comment:
Let's fix the above string as well
`"Spearman correlation; distance = 1 - ρ/2"`